client/index.html

```diff
@@ -301,20 +301,36 @@ <h1 class="title">Settings</h1>
 								</label>
 							</div>
 							{{/if}}
+							{{#unless public}}
+							<div class="col-sm-12">
+								<h2>Push Notifications</h2>
+							</div>
+							<div class="col-sm-12">
+								<button type="button" class="btn" id="pushNotifications" disabled data-text-alternate="Unsubscribe from push notifications">Subscribe to push notifications</button>
+								<div class="error" id="pushNotificationsHttps">
+									<strong>Warning</strong>:
+									Push notifications are only supported over HTTPS connections.
+								</div>
+								<div class="error" id="pushNotificationsUnsupported">
+									<strong>Warning</strong>:
+									<p>Push notifications are not supported by your browser.</p>
+								</div>
+							</div>
+							{{/unless}}
 							<div class="col-sm-12">
-								<h2>Notifications</h2>
+								<h2>Browser Notifications</h2>
 							</div>
 							<div class="col-sm-12">
 								<label class="opt">
 								<input id="desktopNotifications" type="checkbox" name="desktopNotifications">
-								Enable desktop notifications<br>
+								Enable browser notifications<br>
```

Comment:
We decided to keep merging notification types and other UX/UI major improvements for later.